live-examples/js-examples/reflect/reflect-deleteproperty.html

```diff
@@ -12,6 +12,6 @@
 Reflect.deleteProperty(array1, '3');
 
 console.log(array1);
-// expected output: Array [1, 2, 3, , 5]
+// expected output: Array(5) [ 1, 2, 3, undefined, 5 ]
```

Comment:
Again, this comment change is not quite accurate - the result shown in the interactive example's console here is: `Array [1, 2, 3, undefined, 5]`.

Reply:
I have made the changes as requested.